app/src/main/res/layout/content_main.xml

```diff
@@ -27,7 +27,7 @@
             android:layout_marginEnd="@dimen/rocket_image_margin_end"
             android:layout_marginRight="@dimen/rocket_image_margin_end"
             android:onClick="moveRocket"
-            android:src="@drawable/ic_rocket_image" />
+            android:src="@drawable/rocketimage" />
```

Comment:
@RohitJain1103 @iamareebjamal The previous image was a vector drawable asset and would scale accordingly. This image is a simple png. @RohitJain1103 if you can add a vector asset for the same please do so. Otherwise, this is unnecessary. Moreover, this image would behave badly with the background as the white background of this image will stay over the actual background giving a bad effect.

Reply:
> Otherwise, this is unnecessary. Moreover, this image would behave badly with the background as the white background of this image will stay over the actual background giving a bad effect.

@jddeep I think this PR is necessary because the previous image which you added was looking quite unprofessional and naive. I think you should once check the image by clicking on it. The background is transparent (which is I intentionally did). You can even build it and check further you want to validate it further.

> The previous image was a vector drawable asset and would scale accordingly. This image is a simple png.

I have already thought about the scaling thing. I was just waiting for our mentors @CloudyPadmal @mariobehling to approve the image first. If it is approved I would surely make different drawable folders for different resolution screens and place my corresponding png's in all of them. Cheers!

Reply:
No png, please use vector asset

Reply:
@iamareebjamal cool